### PR TITLE
fix unit tests

### PR DESCRIPTION
**Pull request by p0lyn0mial**
## Related issue(s)

Fixes #

### CODE REVIEW

File: pkg/reconciler/tenancy/workspace/workspace_reconcile_scheduling_test.go [hunk 1]
```diff
@@ -158,7 +158,6 @@ func TestSchedulingReconciler(t *testing.T) {
 				},
 			),
 			wantStatus: reconcileStatusContinue,
-		},*/
```
Comment (sttts) [inline pkg/reconciler/tenancy/workspace/workspace_reconcile_scheduling_test.go:161]: is this correct?

File: pkg/reconciler/tenancy/workspace/workspace_reconcile_scheduling_test.go [hunk 2]
```diff
@@ -178,7 +177,6 @@ func TestSchedulingReconciler(t *testing.T) {
 			),
 			wantStatus: reconcileStatusContinue,
 		},
-		// TODO:(p0lyn0mial): fix me
 		/*{
```
Comment (sttts) [inline pkg/reconciler/tenancy/workspace/workspace_reconcile_scheduling_test.go:180]: I think we have to fix these, or rewrite or prune from irrelevant tests.

Comment (p0lyn0mial) [reply pkg/reconciler/tenancy/workspace/workspace_reconcile_scheduling_test.go:180]: after all workspace scheduling changed a lot :)

File: pkg/admission/namespacelifecycle/admission_test.go [hunk 3]
```diff
@@ -18,6 +18,7 @@ package namespacelifecycle
 
 import (
 	"context"
+	tenancyv1beta1 "github.com/kcp-dev/kcp/pkg/apis/tenancy/v1beta1"
```
Comment (sttts) [inline pkg/admission/namespacelifecycle/admission_test.go:21]: order

File: pkg/admission/namespacelifecycle/admission_test.go [hunk 4]
```diff
@@ -77,7 +76,7 @@ func TestAdmit(t *testing.T) {
 		t.Run(tt.name, func(t *testing.T) {
 			handler, err := newWorkspaceNamespaceLifecycle()
 			require.NoError(t, err, "error creating admission plugin")
-			handler.getClusterWorkspace = func(clusterName logicalcluster.Name, name string) (*tenancyv1alpha1.ClusterWorkspace, error) {
+			handler.getClusterWorkspace = func(clusterName logicalcluster.Name, name string) (*tenancyv1beta1.Workspace, error) {
```
Comment (sttts) [inline pkg/admission/namespacelifecycle/admission_test.go:79]: rename to getWorkspace

**Pull request merged**